tests/components/ViewedByCustomer/ViewedByCustomer.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { ViewedByCustomer } from '../../../src/Index';
```

Comment:
What happen if **isUserActionView** is not present on the result ?

Reply:
Will add section for 'falsy' values.